### PR TITLE
Use the new location of the cli module as the entrypoint

### PR DESCRIPTION
**Pull request by vdonato**
Checklist

- [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
- [x] Bumped the build number (if the version is unchanged)

### CODE REVIEW

**Comment by raybellwaves** [comment]
Thanks. Came across this today. @vdonato added you as a maintainer if that's ok. You may have to close and open the PR then you should be able to merge.

**Review by raybellwaves — APPROVED** [review]

**Comment by kmcgrady** [comment]
@raybellwaves Does this apply to 1.12.2 AND 1.12.1 where the issue is introduced?

**Comment by raybellwaves** [comment]
> @raybellwaves Does this apply to 1.12.2 AND 1.12.1 where the issue is introduced?

Think it'll only apply to 1.12.2

**Comment by vdonato** [comment]
Hm, looks like I can't merge, but I think I tried closing and reopening before the change adding me to the maintainers list got reflected. Let me try again.

**Comment by vdonato** [comment]
f

**Comment by raybellwaves** [comment]
> Hm, looks like I can't merge, but I think I tried closing and reopening before the change adding me to the maintainers list got reflected. Let me try again.

Sorry didn't work in my test PR either. I wonder if it takes an hour or so... Would have thought it would have been straight away

**Comment by raybellwaves** [comment]
> Hm, looks like I can't merge, but I think I tried closing and reopening before the change adding me to the maintainers list got reflected. Let me try again.

You may have to accept an e-mail invite as well

**Comment by raybellwaves** [comment]
Thanks for the fix here. Merged the other PR where I ran `@conda-forge-admin, please rerender` may have helped added you as an owner in the future.

**Comment by vdonato** [comment]
No problem! It took me awhile to notice the email with the invite to the organization, so that must have been the issue
